contracts/lease/src/position/mod.rs

```diff
@@ -82,24 +86,37 @@ where
         self.amount -= asset
```

Comment:
[a few lines above] Please update the fn name and location in the line "// for more insights refer to the TODO next to `Liability::check`"